helm/policy-role.yaml

```diff
@@ -0,0 +1,19 @@
+##
```

Comment:
Perhaps move this to `helm/.generate/` so that its more obvious that this file is only used to generate the sample.